Products/ZenModel/GraphPoint.py

```diff
@@ -100,6 +100,17 @@ def manage_editProperties(self, REQUEST):
         validation on fields and either return error message or save
         results.
         '''
+
+        lineType = REQUEST.get('lineType', '').strip().lower()
+        stacked = str(REQUEST.get('stacked', '')).strip().lower()
+        stackedLine = lineType == 'line' and stacked == 'true'
+        if stackedLine:
+            messaging.IMessageSender(self).sendToBrowser(
+                'Invalid definition',
+                'Only Area graphs may be stacked. ',
+                priority=messaging.WARNING
+            )
+
```

Comment:
Does this mean if a graph is already defined as line + stacked, that they're going to get a flare warning every time a page is shown with that graph on it?

Reply:
This is only checked after the [Save] button is pressed.  Existing definitions will continue to show as non-stacked line charts -- the stacked==true is ignored when line is defined.